modules/ocf_backups/files/create-encrypted-backup

```diff
@@ -37,33 +37,37 @@ umask 077
 rm -rf "$ARCHIVE_DIRECTORY"
 mkdir "$ARCHIVE_DIRECTORY"
 
-compress_and_encrypt() {
-    dd if="$SNAPSHOT_PATH" bs=32M | pigz |
-        gpg --homedir "$gpgdir" \
-            --no-tty \
-            --quiet \
-            --encrypt \
-            --trust-model always \
-            --compress-algo none \
-            --recipient ckuehl@berkeley.edu \
-            --recipient willh@ocf.berkeley.edu \
-            --recipient jvperrin@ocf.berkeley.edu \
-            --recipient mattmcal@ocf.berkeley.edu \
-            --recipient kpengboy@ocf.berkeley.edu \
-            --recipient abizer@ocf.berkeley.edu \
-            --recipient benzhang4@gmail.com \
-            --recipient daniel@dkess.me \
-            --recipient cooperc@ocf.berkeley.edu
+compress() {
+    dd if="$SNAPSHOT_PATH" bs=32M | pigz
 }
 
-split_parts() {
-    split -d -a 4 -b 5G - \
-        "${ARCHIVE_DIRECTORY}/backup-$(date +%Y-%m-%d).img.gz.gpg.part"
+# this is the filter that `split` will use to encrypt each .part file after
+# splitting. $FILE is the .part filename, we append .gpg to that.
+encrypt_cmd="gpg --homedir "$gpgdir" \
+                 --no-tty \
+                 --quiet \
+                 --encrypt \
+                 --trust-model always \
+                 --compress-algo none \
+                 --recipient ckuehl@berkeley.edu \
+                 --recipient willh@ocf.berkeley.edu \
+                 --recipient jvperrin@ocf.berkeley.edu \
+                 --recipient mattmcal@ocf.berkeley.edu \
+                 --recipient kpengboy@ocf.berkeley.edu \
+                 --recipient abizer@ocf.berkeley.edu \
+                 --recipient benzhang4@gmail.com \
+                 --recipient daniel@dkess.me \
+                 --recipient cooperc@ocf.berkeley.edu \
```

Comment:
Stretch goal: Feel free to prune the list here too, we haven't heard from some of these people in years and probably only current or reasonably recent people need to be on here. That being said, it's not a big deal, but probably good to update periodically. Keep in mind this list is also only used for new backups and any old backups would continue to be accessible, which seems reasonable to me.